### PR TITLE
Make acis_thermal_check compatible with acisfp_check

### PR DESCRIPTION
**Pull request by jzuhone**
In order to integrate the focal plane temperature model with `acis_thermal_check`, some changes needed to be made, given how different `acisfp_check` is. 

A PR for `acisfp_check` which goes with this PR can be found at acisops/acisfp_check#8.

This PR also adds green dashed lines to validation plots marking perigee passages. 

This PR should not be merged until approval but also after the following tests have been performed:

- [x] Tests of the new `acisfp_check` against old loads
- [x] Testing the other model tools against old loads to make sure there are no changes
- [x] Test all model tools, including `acisfp_check`, against a few weekly loads

### CODE REVIEW

File: acis_thermal_check/main.py [hunk 1]
```diff
@@ -543,6 +508,16 @@ def make_prediction_plots(self, outdir, states, times, temps, load_start):
 
         return plots
 
+    def get_histogram_mask(self, msid, tlm, limit):
+        # Make quantiles. Use the histogram limits to decide
```
Comment (jeanconn) [inline acis_thermal_check/main.py]: This comment isn't helping me remember what this is actually doing.

File: acis_thermal_check/main.py [hunk 2]
```diff
@@ -628,26 +607,27 @@ def make_validation_plots(self, tlm, model_spec, outdir, run_start):
             ax.set_title(msid.upper() + ' validation')
             ax.set_ylabel(labels[msid])
             ax.grid()
+            # add lines for perigee passages
+            for rz in rzs:
+                ptimes = cxctime2plotdate(date2secs([rz.start, rz.stop]))
```
Comment (jeanconn) [inline acis_thermal_check/main.py]: This might be one routine simpler if you used rz.tstart and rz.tstop.

**Comment by jeanconn** [comment]
Thanks @jzuhone

**Comment by jzuhone** [comment]
@jeanconn thank you for pointing it out!

**Pull request merged**